test/chunking-form/index.js

```diff
@@ -62,7 +62,11 @@ describe('chunking form', () => {
 								expectedFiles = [];
 							}
 
-							assert.deepEqual(actualFiles, expectedFiles);
+							assert.deepEqual(Object.keys(actualFiles), Object.keys(expectedFiles));
```

Comment:
To my knowledge, there is no guaranteed order for `Object.keys`. Together with the check below, wouldn't it be enough to check that both have the same length?

Reply:
I believe `Object.keys` is indeed spec'd to enumerate keys in the order they were added.  I believe this was an ES6 addition.

Reply:
Yes object order can be relied on fine, and it's a nicer error message to get a file listing diff than a numeric diff error.

Reply:
In that case, I stand corrected. Sounds like `fixturify` also has a defined order so no problem here i guess.

Reply:
Hi Lukas. So sorry to bother - I know it's Sunday, but is there any chance this will be seeing the light of day soon? I'm eager to continue testing out Rollup's new code splitting 😁

Reply:
And sorry for not getting back to you but I was out hiking yesterday 😉I'm on it now!

Reply:
Goodness please don't aplogize 🙃

Hope the hike was good - I'll look forward to getting back in there whenever this gets published 😬

Reply:
As in—now 😁

Reply:
Woot!  Thanks a ton!